### DISCUSSION TITLE
AudioBuffer Formats

### DISCUSSION THREAD

**Discussion by inst5** [cables-gl/cables_docs#148] [Answers & Questions] [answered]
Hi,
is it possible to upload an OGG Opus file to Audiobuffer? 
As I would like to use multichannel Audio. 
Or is it limited to mp3 and wav?

thanks!

**Comment by steam0r** [comment, marked as answer]
hey,

are you having any problems with ogg/opus files? this seems to work fine for me:

![Bildschirmfoto 2022-02-14 um 10 42 44](https://user-images.githubusercontent.com/151271/153839312-4c59342b-5c9a-43f6-929e-f2cfaa7ffeea.png)


best,
stephan

**Reply by inst5** [reply]
Hi Stephan,
ah great - yes with FileInput it did work. Before I was trying to simply use Audiobuffer and tried to select my .opus from the file select window. But there the .opus did not show up. But anyway with FileInput it works.

thanks!
